packages/next/build/webpack-config.ts

```diff
@@ -1006,6 +1018,15 @@ export default async function getBaseWebpackConfig(
       webpack,
     })
 
+    if (
+      dev &&
+      webpackConfig.devtool &&
```

Comment:
```suggestion
```

Reply:
Why was this change done? Disabling source maps altogether shouldn't cause any performance issues as far as I can tell.